portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -110,6 +121,25 @@ private Query buildQuery(String commentPicId, boolean isDateDescending) {
     return comments;
   }
 
+  /**
+   * Translate comments using the Cloud Translation API
+   */
+  private void translateComments(List<Comment> comments, String targetLang) {
+    try {
+      Translate translate = TranslateOptions.getDefaultInstance().getService();
+      
+      for (Comment comment : comments) {
+        Translation translation = translate.translate(
+            comment.getComment(),
+            Translate.TranslateOption.targetLanguage(targetLang));
+
+        comment.setComment(translation.getTranslatedText());
```

Comment:
Nit: comment.setComment() is a little repetitive. What do you think about comment.setText()?

Reply:
Sure, sounds good!